scijava/scijava-ops-api/templates/main/java/org/scijava/ops/api/OpBuilder.vm

```diff
@@ -652,56 +652,140 @@ public class OpBuilder {
 #end
 		}
 
+		/**
+		 * Match a {@link org.scijava.function.Computers} op that can use the given pre-allocated output instance.
+		 *
```

Comment:
There are even more `@see` blocks here, and no explanation why...

Reply:
Will resolve all these as indicated in https://github.com/scijava/incubator/pull/74#discussion_r1136486448